dask/diagnostics/profile_visualize.py

```diff
@@ -168,9 +169,16 @@ def visualize(profilers, file_path=None, show=True, save=True, **kwargs):
     The completed bokeh plot object.
     """
     bp = import_required('bokeh.plotting', _BOKEH_MISSING_MSG)
-    from bokeh.io import _state
+    import bokeh
 
-    if not _state._notebook:
+    if bokeh.__version__ >= LooseVersion("0.12.10"):
```

Comment:
You want `LooseVersion(bokeh.__version__) >= "0.12.10"`.

Reply:
Whoops... thanks.